modellingworld.Rmd

```diff
@@ -0,0 +1,112 @@
+---
+title: "The ActDev Project"
+subtitle: "an open and scalable platform for simulating new developments and their impacts on walking and cycling"
+author: "Robin Lovelace, Joey Talbot, Nikée Groot, Martin Lucas-Smith, Patrick Johansson, Simon Nutall, Andrew Speakman, Dustin Carlino & Nathanael Sheehan"
```

Comment:
That is a LOT of authors. You could reduce that to put just you and make the point that this is part of a wider project in a subsequent slide. No big deal but could reduce confusion, especially as you'll be the only one who's there!

Reply:
Sure - I will add these names to my notes to make sure everyone gets a shout out!